### PR TITLE
Txn schema enum

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: pkg/libovsdb/schema.go [hunk 1]
```diff
@@ -541,34 +545,110 @@ func (schemas *Schemas) LookupTable(dbname, table string) *TableSchema {
...
-	_, ok := value.(bool)
-	return ok
+	typeval, ok := value.(bool)
+	if !ok {
+		return false
+	}
+	if baseType.Enum == nil {
+		return true
+	}
+	for _, v := range baseType.Enum {
```
Comment (roytman) [inline pkg/libovsdb/schema.go:595]: I don't think that boolean can contain Enum, but it OK to have it just in case

File: pkg/ovsdb/transact.go [hunk 2]
```diff
@@ -565,6 +582,8 @@ func (c *Condition) Compare(row *map[string]interface{}) (bool, error) {
 		return c.CompareString(row)
 	case libovsdb.TypeUUID:
 		return c.CompareUUID(row)
+	case libovsdb.TypeEnum:
```
Comment (roytman) [inline pkg/ovsdb/transact.go:585]: Why do we need to compare Enum with Condition? I think Enum is relevant as a restriction to new values only, isn't it ?

File: pkg/libovsdb/schema.go (hunk 1 above)
Comment (hunchback) [reply pkg/libovsdb/schema.go:595]: removed

File: pkg/ovsdb/transact.go (hunk 2 above)
Comment (hunchback) [reply pkg/ovsdb/transact.go:585]: if enum then you need to check what type it is an compare accordingly (we support only enum strings for now)

**Pull request merged**